process-migration/tag-overrides.yaml

```diff
@@ -0,0 +1,22 @@
+schema_version: 1
+
+# Purpose: Tag overrides are for reliably rebuilding the image per the state of
+#          the branch at the time the tag was created, along with other modules
+#          at reliable and unchanging reference points. Useful for CVE respins.
+#          Warning: Only reliable if referenced via a tag itself!
+#   Usage: cekit build --overrides-file tag-overrides.yaml
+
+modules:
+      repositories:
+          - name: cct_module
+            git:
+                  url: https://github.com/jboss-openshift/cct_module.git
+                  ref: sprint-35
+          - name: rhpam-7-image
+            git:
+                  url: https://github.com/jboss-container-images/rhpam-7-image.git
+                  ref: 7.5.0.GA
+          - name: jboss-kie-modules
+            git:
+                  url: https://github.com/jboss-container-images/jboss-kie-modules.git
+                  ref: rhpam-7.5.0.GA
```

Comment:
we do not have 7.6.0 branch/tag yet, shouldn't be master?

Reply:
@spolti , same response as above. tag-overrides.yaml points to tags (even if they're not created yet, since Prod only uses tag-overrides.yaml after release). branch-overrides.yaml is used when we build against master (for now) or 7.6.x (when that branch is cut).